eppo_client/client.py

```diff
@@ -36,9 +34,10 @@ def __init__(
             callback=config_requestor.fetch_and_store_configurations,
         )
         self.__poller.start()
+        self.__evaluator = Evaluator(sharder=MD5Sharder())
 
     def get_string_assignment(
-        self, subject_key: str, flag_key: str, subject_attributes=dict()
+        self, subject_key: str, flag_key: str, subject_attributes=dict(), default=None
```

Comment:
> RAC/FFC should always have a catch-all allocation

Should it? I don't feel strongly either way.

Anyway, yes being able to handle unexpected errors is something we definitely need (Facebook was down today apparently)

Reply:
We already have an unremovable "Default allocation" in the UI, and I think we should keep that behavior. (The default here still matters when the flag doesn't exist in the RAC.)

Reply:
Cleaning up old threads as I do a final pass. Agreed we don't want to bake that assumption into the SDK. And the discussion on defaults has moved on since this